jenkins/bootstrap_test.py

```diff
@@ -1266,10 +1266,15 @@ def testBootstrapCIYaml(self):
         # TODO(krzyzacy): temp until more jobs to be converted
         whitelist = [
             'kubernetes-e2e-cri-gce',
-            'kubernetes-e2e-gci-gce'
+            'kubernetes-e2e-gci-gce',
+            'kubernetes-e2e-gce[a-z-]+release-1.[345]$'
+        ]
+
+        blacklist = [
+            'kubernetes-e2e-gce-gci-'
         ]
             
-        is_modern = lambda name: any(re.match(w, name) for w in whitelist)
+        is_modern = lambda name: any(re.match(w, name) for w in whitelist) and all(re.match(b, name) is None for b in blacklist) 
```

Comment:
Favor implicit comparisons over `is None`

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations